src/assets/style.scss

```diff
@@ -224,6 +232,16 @@ details summary {
   cursor: pointer;
 }
 
+summary h1,
+summary h2,
+summary h3,
+summary h4,
+summary h5,
+summary h6 {
+  display: inline-block;
+  margin: 0;
+}
```

Comment:
This is needed so I can use headings in the sidebar within `<summary>` elements (the expandable drop-downs) without them having huge margins like they're supposed when in the actual article itself.